src/app/pages/HomePage/Graph/Graph/index.tsx

```diff
@@ -71,6 +117,90 @@ const preventDoubleClick: { onDoubleClick: MouseEventHandler } = {
   onDoubleClick: e => e.preventDefault(),
 }
 
+const handleHover: (
+  layer: Layer,
+  set: (layer: Layer | null) => void,
+) => {
+  onMouseEnter: MouseEventHandler
+  onMouseLeave: MouseEventHandler
+} = (layer: Layer, set: (layer: Layer | null) => void) => ({
+  onMouseEnter: () => {
+    set(layer)
+  },
+  onMouseLeave: () => {
+    set(null)
+  },
+})
+
+const GraphParaTimeStatus: FC<
+  PropsWithChildren<{
+    fillText: string
+    iconX: number
+    iconY: number
+    textX: number
+    textY: number
+    outOfDate?: boolean
+  }>
+> = ({ children, fillText, iconX, iconY, textX, textY, outOfDate }) => {
+  const { t } = useTranslation()
+
+  if (outOfDate === undefined) {
+    return (
+      <text x={textX} y={textY} fill={fillText} fontSize="12px">
+        {children}
+      </text>
+    )
+  }
+
+  return (
+    <>
+      <text x={textX} y={textY} fill={fillText} fontSize="12px">
+        {children}
+        {!outOfDate && (
+          <tspan dx="-40" dy="12" fontSize="7px">
+            {t('home.online')}
+          </tspan>
+        )}
+        {outOfDate && (
+          <tspan dx="-40" dy="12" fontSize="7px">
+            {t('home.offline')}
+          </tspan>
+        )}
+      </text>
+      <g transform={`translate(${iconX}, ${iconY}) scale(0.4)`}>
+        {!outOfDate && (
+          <path
+            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
+            fill={COLORS.eucalyptus}
+          />
+        )}
+        {outOfDate && (
+          <path
+            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
+            fill={COLORS.error}
+          />
+        )}
+      </g>
+    </>
+  )
+}
+
+const LayerStatus: FC<{ scope: SearchScope; statusChange: (outOfDate?: boolean) => void }> = ({
```

Comment:
I don't think this makes sense as a component

Reply:
Right, the issue here is that I only want to trigger `useRuntimeFreshness` for certain layers. Kind of hard to do otherwise as conditional component. Do you have any ideas how to better resolve this?

Reply:
Ah, useRuntimeFreshness doesn't have {enabled: false} :shrug: